include/librealsense2/h/rs_device.h

```diff
@@ -130,23 +130,6 @@ void rs2_loopback_disable(const rs2_device* device, rs2_error** error);
 */
 int rs2_loopback_is_enabled(const rs2_device* device, rs2_error** error);
 
-/**
```

Comment:
I'd prefer to keep the APIs, to not break any existing code. 
At runtime these APIs check if controllers are available and on the T265 will always return false